src/IonUnicode.ts

```diff
@@ -13,6 +13,11 @@
  * permissions and limitations under the License.
  */
 
+import { TextDecoder } from "util";
+
+// verify if TextDecoder exists globally or not
```

Comment:
Nit: this comment is entirely accurate, but I think it would be more helpful to explain the higher-level purpose of the code it accompanies.

How about:
```suggestion
// Check whether this runtime supports the `TextDecoder` feature
```
?